torch/_inductor/mkldnn.py

```diff
@@ -138,7 +130,7 @@ def _update_module_params(self, conv, unary, input_size):
 
     def _conv_forward(self, input, weight, bias):
         if self.padding_mode != "zeros":
-            return torch.ops.mkldnn._convolution_pointwise(
+            return torch.mkldnn_convolution(
```

Comment:
Let's add a single `conv` and linear for the repacked `conv`. The semantic of `_conv_pointwise` is confusing as it serves both the single conv and the fused conv.